src/prisma.ts

```
@@ -1,32 +1,27 @@
-  prismaGlobal: ReturnType<typeof prismaClient>;
-} & typeof global;
+// if using planetscale db, use this. if not, comment out and uncomment the local db setup below
+const adapter = new PrismaPlanetScale({ url: process.env.DATABASE_URL, fetch });
+const prismaClient = new PrismaClient({ adapter, omit: omitConfig });
 
-export const prisma = globalThis.prismaGlobal ?? prismaClient();
+// if using local db, uncomment this and comment out the above
+// const datasourceUrl = process.env.LOCAL_DATABASE_URL;
+// const prismaClient = new PrismaClient({ datasourceUrl, omit: omitConfig });
+
+declare const globalThis: { prismaGlobal: typeof prismaClient } & typeof global;
```

Biome (1.9.4)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)